vowpalwabbit/slates.h

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) by respective owners including Yahoo!, Microsoft, and
+// individual contributors. All rights reserved. Released under a BSD (revised)
+// license as described in the file LICENSE.
+
+#pragma once
+
+#include "reductions_fwd.h"
+
+#include "slates_label.h"
+#include "learner.h"
+
+#include <string>
+#include <vector>
+
+namespace slates
+{
+struct slates_data
+{
+ private:
+  std::vector<slates::label> _stashed_labels;
+
+  /*
+  The primary job of this reduction is to convert slate labels to a form CCB can process.
```

Comment:
Can you add a comment stating shared must come before action, which must come before slot in the example vector?